editor/editor_inspector.cpp

```diff
@@ -1563,14 +1566,17 @@ void EditorInspectorCategory::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
-			if (menu) {
-				if (is_favorite) {
-					menu->set_item_icon(menu->get_item_index(EditorInspector::MENU_UNFAVORITE_ALL), get_editor_theme_icon(SNAME("Unfavorite")));
-				} else {
-					menu->set_item_icon(menu->get_item_index(MENU_OPEN_DOCS), get_editor_theme_icon(SNAME("Help")));
-				}
+			menu_icon_dirty = true;
+			_update_icon();
+		} break;
+
+		case NOTIFICATION_TRANSLATION_CHANGED: {
```

Comment:
The editor will update everything on translation change, so this is probably unnecessary.

Reply:
The Favorites category won't be not updated. EditorInspector creates only one such instance.